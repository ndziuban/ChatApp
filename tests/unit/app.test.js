import React from 'react';
import View from '../../app/page/view';
import renderer from 'react-test-renderer';

it('View (initial) renders correctly', () => {
  const tree = renderer
    .create(
      <View />
    )
    .toJSON();
  expect(tree).toMatchSnapshot();
});
