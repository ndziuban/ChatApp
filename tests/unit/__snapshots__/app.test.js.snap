// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`View (initial) renders correctly 1`] = `
<div
  className="app__container"
>
  <div
    className="chat-window"
  >
    <div
      className="chat-window__header"
    >
      <div
        className="header-content"
      >
        <span>
          Laura
        </span>
        <span
          className="header-content__subtext"
        >
          Active
        </span>
      </div>
    </div>
    <div
      className="overflow-content"
    >
      <div
        className="chat-window__message-container"
      >
        <div />
      </div>
    </div>
    <div
      className="input__container"
    >
      <textarea
        className="input"
        onChange={[Function]}
        onKeyDown={[Function]}
        placeholder="Type here to send a message..."
        rows="2"
        value=""
      />
    </div>
  </div>
  <div
    className="app__divisor"
  />
  <div
    className="chat-window"
  >
    <div
      className="chat-window__header"
    >
      <div
        className="header-content"
      >
        <span>
          Rob
        </span>
        <span
          className="header-content__subtext"
        >
          Active
        </span>
      </div>
    </div>
    <div
      className="overflow-content"
    >
      <div
        className="chat-window__message-container"
      >
        <div />
      </div>
    </div>
    <div
      className="input__container"
    >
      <textarea
        className="input"
        onChange={[Function]}
        onKeyDown={[Function]}
        placeholder="Type here to send a message..."
        rows="2"
        value=""
      />
    </div>
  </div>
</div>
`;
